plugins/in_calyptia_fleet/in_calyptia_fleet.c

```diff
@@ -485,40 +485,6 @@ static void *do_reload(void *data)
     return NULL;
 }
 
-static int test_config_is_valid(struct flb_in_calyptia_fleet_config *ctx,
-                                flb_sds_t cfgpath)
-{
-    return FLB_TRUE;
```

Comment:
`test_config_is_valid` is a no-op in practice and according to @pwhelan we'll never be ably to fully validate up front. Removing this function.